assembly/std/index.ts

```diff
@@ -60,11 +70,14 @@ export function localCall(
  *
  * Note: arguments serialization is to be handled by the caller and the callee.
  *
- * @param bytecode -
- * @param functionName -
- * @param args -
+ * @param bytecode - The bytecode of the contract containing the function to execute.
+ * @param functionName - The name of the function to call in that contract.
+ * @param args - The arguments of the function we are calling.
  *
- * @returns function returned value
+ * @remarks
+ *  Runtime exception if the function doesn't exist in the bytecode !
```

Comment:
same here, I think it would be better to put it a @throw tag, I wouldn't put an exclamation mark at the end.

Reply:
Maybe we can check if there is more exeption like if the function doesn't exist.